docs/index.rst

```diff
@@ -77,6 +80,20 @@ Now you can connect to port 80 of that ip address. Here's an example
     so you might find this ip address changes over time. You can always find out
     what ip is being used with ``docker-machine ip [machine name]``
 
+You may need to use a reliable hostname to access your container server (e.g. for
+Firefox Accounts). You can set one by editing your ``/etc/hosts`` file on your
+native operating system. For example::
+
+    192.168.99.100  olympia.dev
```

Comment:
Yeah you can, I've tended to not bother with hostnames because of it. There's got to be a better solution with some kind of discovery service so these things stay in-sync.